### DISCUSSION TITLE
Database ER diagram

### DISCUSSION THREAD

**Discussion by DrMichael** [teslamate-org/teslamate#1224] [Show and tell]
Hi all,

I produced an ER diagram for the database. Perhaps helpful.

Michael
![teslamate - ER diagram](https://user-images.githubusercontent.com/8087682/105055447-5e042180-5a73-11eb-85ec-7e7ef3798dc1.png)

**Comment by OzBrown** [comment]
This helps a lot in understanding how the code works across the Graphs and is read from the API Data :)  

Really great work

**Reply by bahree** [reply]
May be we can add a developer section in the docs and have this there?

**Comment by jdklii22** [comment]
> Hi all,
> 
> I produced an ER diagram for the database. Perhaps helpful.
> 
> Michael ![teslamate - ER diagram](https://user-images.githubusercontent.com/8087682/105055447-5e042180-5a73-11eb-85ec-7e7ef3798dc1.png)

Thanks for sharing this. The visualziation really helps!
